docs/destinations/forwarding-events/snowbridge/configuration/index.md

```diff
@@ -27,9 +27,9 @@ There’ll be no external statistics reporting or sentry error reporting.
 
 ## License
 
-Since version 2.4.0, Snowbridge is released under the [Snowplow Limited Use License](https://docs.snowplow.io/limited-use-license-1.0/) ([FAQ](/docs/contributing/limited-use-license-faq/index.md)).
+Since version 2.4.1, Snowbridge is released under the [Snowplow Limited Use License](https://docs.snowplow.io/limited-use-license-1.0/) ([FAQ](/docs/contributing/limited-use-license-faq/index.md)).
```

Comment:
```suggestion
Since version 2.4.0, Snowbridge is released under the [Snowplow Limited Use License](https://docs.snowplow.io/limited-use-license-1.0/) ([FAQ](/docs/contributing/limited-use-license-faq/index.md)).
```